sql/src/test/java/io/crate/analyze/MetaDataToASTNodeResolverTest.java

```diff
@@ -96,6 +96,7 @@ public void testBuildCreateTableColumns() throws Exception {
                      "   \"mapping.total_fields.limit\" = 1000,\n" +
                      "   max_ngram_diff = 1,\n" +
                      "   max_shingle_diff = 3,\n" +
+                     "   \"merge.scheduler.max_thread_count\" = 4,\n" +
```

Comment:
I think this will be a bit problematic since the value depends on the environment where the test is running on. E.g. on our CI systems it will be lower.

Maybe we can make the assertion a bit more lenient in that regard or fix the value in the `create table` statement, so that the tests are deterministic on all environments?

Reply:
Right, I will fix this value.